.circleci/config.yml

```diff
@@ -68,7 +68,7 @@ jobs:
       - run:
           name: Assemble
           command: |
-            ./gradlew --no-daemon --parallel clean spotlessCheck compileJava compileTestJava assemble
```

Comment:
Let's keep `--no-daemon` - daemons are useless in ci docker images.

Reply:
Are these changes still requested, given that Gradle specially recommends the daemon in CI?
Should I add --no-daemon only to JMH?
At any rate I can confirm that acceptance tests work well without the daemon (and I'd say faster, but I haven't real measurements). And this build did finish correctly.